data/helpers.d/utils

```diff
@@ -487,3 +485,62 @@ ynh_check_app_version_changed () {
   fi
   echo $return_value
 }
+
+# Compare the current package version against another version given as an argument.
+# This is really useful when we need to do some actions only for some old package versions.
+#
+# example: ynh_compare_package_version --comparison gt --version 2.3.2~ynh1
+# This example will check if the installed version is greater than (gt) the version 2.3.2~ynh1
+#
+# usage: ynh_compare_package_version --comparison lt|gt|le|ge
+# | arg: --comparison - Comparison type. Could be : le (lower than), gt (greater than), le (lower or equal), ge (greater or equal)
+# | arg: --version - The version to compare. Need to be a version in the yunohost package version type (like 2.3.1~ynh4)
+#
+# Return 0 if the evaluation is true. 1 if false.
+#
+# Requires YunoHost version 3.8.0 or higher.
+ynh_compare_package_version() {
```

Comment:
Considering the way the helper work
`ynh_compare_current_package_version` sound more clear to me.
I had a hard time figure out what was the expecting result of this helper.